src/definitions/modules/dropdown.js

```diff
@@ -3397,7 +3397,7 @@ $.fn.dropdown = function(parameters) {
             return $(event.target).closest($icon).length > 0;
           },
           chrome: function() {
-            return !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime);
+            return !!window.chrome && !window.StyleMedia;
```

Comment:
`VM180:1 Uncaught ReferenceError: StyleMedia is not defined
    at <anonymous>:1:1` for my chrome.
Perhaps `window.styleMedia` ?

Reply:
No, because we explicitely check for old Edge here (which also has `window.chrome` but isn't chrome!)
The check for Uppercase `StyleMedia` is only available in old Edge (while lowercase is available on all modern browsers)
### Old Edge
![image](https://user-images.githubusercontent.com/18379884/157066078-4ee26e5c-96fe-495b-bd48-5d1e507aafd0.png)